lib/v4/migration-helpers/__mocks__/mock-fs.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
```

Comment:
What I meant is that if you name your mock file with the name of the module (here `__mocks__/fs-extra`), and then in any other file call only `jest.mock('fs-extra')`, jest will pick up this mock file automatically.

See https://jestjs.io/docs/manual-mocks#mocking-node-modules

Reply:
oh nice I didn't know that! thanks